pkg/alertmanager/multitenant.go

```diff
@@ -687,10 +697,11 @@ func (am *MultitenantAlertmanager) syncConfigs(ctx context.Context, cfgMap map[s
 	}
 
 	userAlertmanagersToStop := map[string]*Alertmanager{}
-
 	am.alertmanagersMtx.Lock()
 	for userID, userAM := range am.alertmanagers {
-		if _, exists := cfgMap[userID]; !exists {
+		_, exists := cfgMap[userID]
+		_, initSkipped := amInitSkipped[userID]
```

Comment:
If the skipped Alertmanager was already running, I'm deleting its data and adding it to the map of Alertmanagers to stop.